src/js/modules/accounts/components/forms/SignOutForm.js

```diff
@@ -28,7 +22,7 @@ const styles = {
   },
 };
 
-const SignOutForm = ({ classes, handleSubmit, submitting, status }) => {
+const SignOutForm = ({ classes, handleSubmit, submitting }) => {
```

Comment:
I'm curious. Why is this a form? Why not just make a button that unsubscribes?

Reply:
A button must be part of a form. It's a whole ting.